### DISCUSSION TITLE
Seeking help on Plasma-flow simulation

### DISCUSSION THREAD

**Discussion by Badtreater** [PlasmaPy/PlasmaPy#2235] [Simulations]
### Description

**Dear developers and fraternities,** 
thank you so much for providing the open-source PLASMA simulation codes. It is my honor to write this letter to you guys who are expanding into new fields. I am a novice in PLASMA research and working hard to complete my doctoral studies.However, due to my poor ability on programming of finite element solvers, it has been hard to complete the simulation code of PLASMA completely on my own. To be honest, I was very excited when I first saw **PlasmaPy**. I thought it might help me to supplement my lack of simulation.
To get back to the point, I hope to use **PlasmaPy** to complete the simulation work in the literature[1], but unfortunately, due to a lack of understanding of the underlying solver, I am unable to make a judgment on whether **PlasmaPy** can correctly complete this simulation process. **I am eager to receive your answers on the applicability of the **PlasmaPy** underlying solver to the simulation of Plasma flow control actuators or suggestions for Plasma flow simulation.** If the work can proceed, I also hope to add colleagues who provide constructive suggestions to the article's signature in my own paper when publishing articles in the future. Thank you so much. 

Reference
[1]Kunwar Pal Singh, Subrata Roy; Modeling plasma actuators with air chemistry for effective flow control. Journal of Applied Physics 15 June 2007; 101 (12): 123308. https://doi.org/10.1063/1.2749467

**Comment by namurphy** [comment]
Very nice to meet you!  Since PlasmaPy is a toolkit for plasma research and education rather than a plasma simulation code, it wouldn't be able to be used to simulate plasma flow control actuators or plasma flows.  If you'd like to learn more about PlasmaPy, please feel free to check out our [online documentation](https://docs.plasmapy.org).  My background is primarily in heliophysics & astronomy, so I'm not very familiar with the codes used to model plasma actuators.  The only one I can think of is [MOOSE](https://mooseframework.inl.gov/), though I'm not familiar enough with its capabilities. I'll leave this thread open in case anyone else has suggestions.

**Reply by Badtreater** [reply]
Thank you so much
